### PR TITLE
feat(obligation): Rename product obligation to project obligation

### PR DESCRIPTION
**Pull request by JaideepPalit**
> Rename product obligation to project obligation
> * Which issue is this pull request belonging to and how is it solving it? (*#998*)
> * Did you add or update any new dependencies that are required for your change? - No

> Have you implemented any additional tests? - No

### Checklist
Must:
- [X] All related issues are referenced in commit messages and in PR

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>

### CODE REVIEW

File: backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxGenerator.java [hunk 1]
```diff
@@ -831,22 +831,22 @@ private void fillOrganisationObligationsTable(XWPFDocument document, Project pro
...
+        });
         setTableBorders(table);
     }
 
     private void fillProjectSpecificObligationsTable(XWPFDocument document, Project project) throws TException {
         XWPFTable table = document.getTables().get(PROJECT_OBLIGATIONS_TABLE_INDEX);
         final int[] currentRow = new int[] { 0 };
 
-        SW360Utils.getOrganisationObligations(project).entrySet().stream().forEachOrdered(oblig -> {
+        SW360Utils.getProjectObligations(project).entrySet().stream()
```
Comment (smrutis1) [inline backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxGenerator.java:848]: indentation

File: frontend/sw360-portlet/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -870,6 +870,7 @@ project.manager=Project manager
 project.name=Project name
 project.name.first.letters=Project Name (first letters)
 project.namelinks=Project NameLinks
+project.obligation=Project Obligation
```
Comment (smrutis1) [inline frontend/sw360-portlet/src/main/resources/content/Language.properties:873]: There is already a key with name `project.obligation`. Same for `Language_vi.properties`

**Review by smrutis1 — CHANGES_REQUESTED** [review]

File: backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxGenerator.java (hunk 1 above)
Comment (JaideepPalit) [reply backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxGenerator.java]: done

File: frontend/sw360-portlet/src/main/resources/content/Language.properties (hunk 2 above)
Comment (JaideepPalit) [reply frontend/sw360-portlet/src/main/resources/content/Language.properties]: removed duplicate

**Review by smrutis1 — APPROVED** [review]
Code looks good to me.

**Comment by JaideepPalit** [comment]
Resolved merge conflict

**Review by smrutis1 — APPROVED** [review]
Tested Successfully on stage.

**Pull request merged**